### ISSUE TITLE
Can't uninstall Pandoc

### ISSUE THREAD

**Issue by MohammedAl-Mahdawi** [jgm/pandoc#9167]
I installed pandoc `3.1.8` days ago on MacBook Pro 2021 Sonoma 14.1 using PKG. Now I want to uninstall it, but I can't I get `Pandoc not installed.` when I run `perl uninstall-pandoc.pl`, but it is installed and working just fine!

**Comment by jgm** [comment]
I just tried it and it worked for me.  Here's the output I got:
```
The following files will be deleted:

/._usr
/usr/._local
/usr/local/._bin
/usr/local/._share
/usr/local/bin/._pandoc
/usr/local/bin/pandoc
/usr/local/share/._man
/usr/local/share/man/._man1
/usr/local/share/man/man1/._pandoc-lua.1
/usr/local/share/man/man1/._pandoc-server.1
/usr/local/share/man/man1/._pandoc.1
/usr/local/share/man/man1/pandoc-lua.1
/usr/local/share/man/man1/pandoc-server.1
/usr/local/share/man/man1/pandoc.1

Do you want to proceed and uninstall pandoc (Y/N)?Y
File /._usr does not exist.  Skipping.
File /usr/._local does not exist.  Skipping.
File /usr/local/._bin does not exist.  Skipping.
File /usr/local/._share does not exist.  Skipping.
File /usr/local/bin/._pandoc does not exist.  Skipping.
Password:
Deleted /usr/local/bin/pandoc
File /usr/local/share/._man does not exist.  Skipping.
File /usr/local/share/man/._man1 does not exist.  Skipping.
File /usr/local/share/man/man1/._pandoc-lua.1 does not exist.  Skipping.
File /usr/local/share/man/man1/._pandoc-server.1 does not exist.  Skipping.
File /usr/local/share/man/man1/._pandoc.1 does not exist.  Skipping.
Deleted /usr/local/share/man/man1/pandoc-lua.1
Deleted /usr/local/share/man/man1/pandoc-server.1
Deleted /usr/local/share/man/man1/pandoc.1
Forgot package 'net.johnmacfarlane.pandoc' on '/'.
Pandoc has been successfully uninstalled.
```
What output did you get?  And are you sure you didn't have another version of pandoc in your path?  What does `which pandoc` say?

@DanStaal I was curious about the paths like `._pandoc` -- why are these showing up?

**Comment by DanStaal** [comment]
The `._*` files would be metadata files, if they exist.  The Finder (and I think some other processes) will create them if the folder is ever opened.  `pkgutil` likely includes them in the list automatically to make sure to catch them if they exist, as they can be created in the background without the user being aware of them, so it's best to assume they exist just in case.

**Comment by MohammedAl-Mahdawi** [comment]
Thank you, dears, for the prompt reply.
The output of `which pandoc` is `/usr/local/bin/pandoc`, and the output of `perl uninstall-pandoc.pl` is `Pandoc not installed`.

**Comment by jgm** [comment]
Odd.  I don't have Sonoma yet so I can't test on that platform.  Perhaps something changed in how packages are handled?

**Comment by jgm** [comment]
If you do `pkgutil --packages`, do you see a list of packages?  Is pandoc in that list?

**Comment by MohammedAl-Mahdawi** [comment]
I see a list of packages, but pandoc is not there.

**Comment by jgm** [comment]
On my system, the following command lists the (real) files installed by pandoc:
```
pkgutil --only-files --files net.johnmacfarlane.pandoc | grep -v _ | sed -e 's/^/\//'
```

**Comment by jgm** [comment]
If pandoc isn't there, then it seems that you didn't install it using the package? At least that's what macOS thinks...

**Comment by jgm** [comment]
Try installing again using the package (.pkg) and see if it then appears in the list.

**Comment by MohammedAl-Mahdawi** [comment]
I installed it again using PKG, and then I could uninstall it using `perl uninstall-pandoc.pl`.

Everything I have on this Mac was imported from another Mac, so maybe this is the issue?!

I mean, I formatted this Mac and used the Migration Assistant app to migrate everything from my other Mac that contains this application among everything else that I need, and I never faced any issues!

Now I don't have pandoc on my machine, so thank you, dears; much appreciated!
